src/components/ExampleCoupon/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useRef, useState } from "react";
+import "./style.css";
+import { Card, Typography } from "antd";
+import { LinkOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import defaultRestaurantLogo from "../../images/defaultRestaurantLogo.jpg";
+
+const { Meta } = Card;
+const { Text, Title } = Typography;
+
+const ExampleCoupon = ({ orgName, orgLink, orgDescription, logoURL }) => {
+  const { t } = useTranslation();
+  const [price, setPrice] = useState("25₺");
+  const timer = useRef(null);
+  const priceList = ["25₺", "50₺", "75₺", "100₺", "150₺", "200₺"];
+  const counter = useRef(-1);
+
+  const changePrices = () => {
+    if (counter.current < 5) {
+      counter.current += 1;
+      setPrice(priceList[counter.current]);
+    } else {
+      counter.current = -1;
+    }
+    if (timer.current === null) {
+      timer.current = setInterval(changePrices, 2000);
+    }
+  };
+
+  const stopTimer = () => {
+    if (timer.current !== null) {
+      clearInterval(timer.current);
+      timer.current = null;
+    }
+  };
+
+  return (
+    <>
+      <Card
+        onMouseEnter={changePrices}
```

Comment:
what suppose to happen when someone hover on the card ?

Reply:
ExampleCoupon, shows every price when hovered on the card every two seconds. Like I mentioned in the comment I made when I pushed.